client/components/createButton/createButton.html

```diff
@@ -0,0 +1,6 @@
+
+<link rel="stylesheet" href="../button/button.css">
+
+<button class="button primary heading-h5" id="buttonPrimary">Create refund</button>
+
+<script src="./index.mjs" type="module"></script>
```

Comment:
You don't need to create separate component for Create button. We already have Button component which also covers this case. If you had task to create such button, either task is incorrect or maybe request was to make "Create" button within page layout and to make it open page with Refund creation form.